_extras/guide.md

```diff
@@ -126,6 +120,8 @@ pmra_poly, pmdec_poly = np.transpose(pm_vertices)
 
 * When you get to `make_dataframe`, you might want to copy and paste it from the notes, rather than retyping.
 
+* Learners may ask why we're using `dict(key1=value1, key2=value2)` rather than `{key1:value1, key2:value2}`. We are using the `dict` syntax so that the key value pairs look like keyword arguments. This may simplify the understanding for learners who are less familiar with dictionaries.
```

Comment:
```suggestion
* Learners may ask why we are using `dict(key1=value1, key2=value2)` rather than `{key1:value1, key2:value2}`. We are using the `dict` syntax so that the key value pairs look like keyword arguments. This may simplify the understanding for learners who are less familiar with dictionaries.
```